public/config/env-config.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2017-present “Neo4j, Inc.” [http://neotechnology.com]
+ *
+ * This file is part of Neo4j Solutions Accelerators.
+ * Neo4j accelerator frameworks and use case based starter kits are Neo4j IP: you cannot redistribute it
+ * and they fall under the terms of the Neo4j Partner Agreement as negotiated by
+ * Neo4j, Inc. and its Partners.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * You are requested to contribute any modifications back to the Neo4j Solutions Team at solutions@neo4j.com
+ *
+ * For questions reach our to Neo4j Solutions Team - solutions@neo4j.com
+ * https://github.com/Neo4jSolutions
+ */
+
+/*
+    // prod
+    REACT_APP_AUTH_DOMAIN: "neo4j-sync.auth0.com",
+    REACT_APP_AUTH_CLIENT_ID: "WI1g67RXNvycmOh9rjTNwfRcKRx0FxhL",
+
+    // dev
+    REACT_APP_AUTH_DOMAIN: "dev-o8s8b52y.auth0.com",
+    REACT_APP_AUTH_CLIENT_ID: "3OOrQh3TYyNOT8Y0AeE2tWek3wX0PAef",
+*/
+
+window._dynamicEnv_ = {
```

Comment:
Can we move all Hive specific logic into the `/src/extensions/hive` folder?

Or does this file explicitly need to be included directly into the HTML?

Reply:
or perhaps just load these values from an `.env` file instead of coding it in `env-config.js`?